great_expectations/checkpoint/actions.py

```diff
@@ -236,13 +236,18 @@ def _run(
 
         validation_success = validation_result_suite.success
         data_docs_pages = None
-
         if payload:
             # process the payload
             for action_names in payload.keys():
                 if payload[action_names]["class"] == "UpdateDataDocsAction":
                     data_docs_pages = payload[action_names]
 
+        # Assemble complete GX Cloud URL for a specific validation result
+        cloud_id = getattr(validation_result_suite_identifier, "cloud_id", None)
+        cloud_url = None
+        if cloud_id:
+            cloud_url = f"https://app.greatexpectations.io/?validationResultId={validation_result_suite_identifier.cloud_id}"
```

Comment:
Instead of `cloud_url` can we call this `validation_result_url` so it can be set in non-Cloud contexts similarly? We try to avoid the `getattr` call usually so I would do something like:
```
validation_result_url: Optional[str] = None
if isinstance(validation_result_suite_identifier, GXCloudIdentifier) and validation_result_suite_identifier.cloud_id:
    validation_result_url = f"https://app.greatexpectations.io/?validationResultId={validation_result_suite_identifier.cloud_id}"
elif base_url:
    # I would need to dig for the logic, but we should be able to get this working for OSS as well
    validation_result_url = f"{base_url}/{TYPICAL OSS ValidationResultIdentifier URL PATH}"
```

Reply:
Also you should add the default Cloud `base_url` to `great_expectations/data_context/cloud_constants.py` and import it in the two files where it's needed.